docs/release-notes-7.1.0/master-docinfo.xml

```diff
@@ -0,0 +1,16 @@
+<title>Release Notes</title>
+<productname>{DocInfoProductName}</productname>
+<productnumber>{DocInfoProductNumber}</productnumber>
+<subtitle>New features, known issues, and resolved issues</subtitle>
+<abstract>
+    <para>
+        {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift through the user interface.
```

Comment:
Add {ocp-first} and {ocp-short}

Reply:
Just to confirm what is this attribute for?.. :-D
I'm just not aware...